session/session_test.go

```diff
@@ -0,0 +1,176 @@
+package session
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"reflect"
+	"testing"
+)
+
+func TestGetDeveloperPortalDataFromEnv(t *testing.T) {
+	// If the BITRISE_PORTAL_DATA_JSON env is already set (e.g for local testing the step),
+	// then we don't want to modify it during the test
+	var envAlreadyPresented bool
```

Comment:
Could use this so no else block is needed:
envAlreadyPresented := true
if ... {
 envAlreadyPresented = false
}

Reply:
Fixed in:
https://github.com/bitrise-io/steps-deploy-to-itunesconnect-deliver/pull/61/commits/ba4be9a32c9c3752a21b80ac930eee7c0fd2fdb6